tools/fluent/basic_syntax.md

```diff
@@ -0,0 +1,131 @@
+# Introduction to Fluent syntax
+
+<!-- toc -->
+
+## Basic syntax
+
+A *message* represents the basic unit of translation in Fluent. This is the simplest form a message could take:
+
+```PROPERTIES
+welcome-msg = Hello, World!
+```
+
+`welcome-msg` is the message identifier (or message ID), and `Hello, World!` is the **value** of this message.
+
+A message can contain **external arguments**, for example:
+
+```PROPERTIES
+welcome-msg = Welcome { $user }
+```
+
+`$user` is the name of the argument, and should never be translated. The fragment included between curly braces is called a **placeable**, and can be moved within the text. For example, the string above would be translated in Italian as follows, leaving the placeable unchanged:
+
+```PROPERTIES
+welcome-msg = Benvenuto { $user }
+```
+
+Messages can also reference other messages. In this case, the placeable won’t include the `$` sign, simply the message identifier:
+
+```PROPERTIES
+menu-settings = Settings
+help-menu-settings = Click “{ menu-settings }” to save the file.
+```
+
+There is a special category of messages, called **private messages**:
+* Private messages have identifiers starting with a dash, e.g. `-brand-short-name`.
+* They can’t be called directly in the source code, but only referenced within other messages.
```

Comment:
There's one more super-important point to be made about private messages: their attributes are entirely managed by the localizer. That's the _private_ part. There can be no attributes, one or many; regardless of how many of them the source language has. Perhaps we could mention it here and then expand on it in the section about attributes?